exporter/alertmanagerexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Alertmanager Exporter
```

Comment:
The readme seems to be missing the auto-generated parts from the mdatagen. Please, review this section from the contribution guidelines: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components

Reply:
Hello! We created a new PR with just the Factory implementation and configs here - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27836. 
Is it okay if I address these suggestions in the other PR and close this one? Kindly suggest, thanks.